source/Camera/Basic/CaptureTutorial.md

```diff
@@ -0,0 +1,121 @@
+## Introduction
+
+This tutorial describes how to use Zivid SDK to capture point clouds and 2D images.
+
+1. The fastest way to get your first capture with the Zivid SDK!
+2. How can you configure the settings of the Zivid camera?
+3. More on captures - HDR and 2D
+4. Bonus!
+
+### Prerequisites
+
+You should have installed Zivid SDK and C++ samples. For more details see [Instructions][installation-instructions-url].
+
+## Part 1 - Quick Start
+
+### Connect
+
+First we have to connect to the camera ([go to source][connect-url]).
+```cpp
+auto camera = zivid.connectCamera();
+```
+
```

Comment:
I would add another line to show how to connect to a specific camera via a serial number.